bin/crew

```diff
@@ -4,6 +4,7 @@ require_relative '../lib/color'
 # Disallow sudo
 abort 'Chromebrew should not be run as root.'.lightred if Process.uid == 0
 
+require 'thread'
```

Comment:
`Thread` is a core module so we don't need to require it

Reply:
No longer threading! so removed! :)